docs/architecture.md

```
@@ -244,10 +251,10 @@ flowchart LR
 
 Server-side git repository using isomorphic-git for repo operations and git CLI for HTTP smart protocol.
 
-**Startup:**
+**Initialization (lazy, on the first git HTTP request):**
 
 ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)